src/EmbedPreview.tsx

```diff
@@ -0,0 +1,384 @@
+                <Select id="panel" name="panel" size="sm" width="auto">
+                  {Object.values(EmbedPanel).map((panel) => (
+                    <option key={panel} value={panel}>
+                      {panel}
+                    </option>
+                  ))}
+                </Select>
+              </FormControl>
+
+              <FormControl
+                display="flex"
+                alignItems="center"
+                justifyContent="space-between"
+              >
+                <FormLabel
+                  marginBottom="0"
+                  htmlFor="readOnly"
+                  whiteSpace="nowrap"
+                >
+                  Editor readonly
+                </FormLabel>
+                <Switch defaultChecked={true} id="readOnly" name="readOnly" />
+              </FormControl>
+
+              <FormControl
+                display="flex"
+                alignItems="center"
+                justifyContent="space-between"
+              >
+                <FormLabel
+                  marginBottom="0"
+                  htmlFor="showOriginalLink"
+                  whiteSpace="nowrap"
+                >
+                  Show original link to visualizer
+                </FormLabel>
+                <Switch
+                  defaultChecked={true}
+                  id="showOriginalLink"
+                  name="showOriginalLink"
+                />
+              </FormControl>
+
+              <FormControl
+                display="flex"
+                alignItems="center"
+                justifyContent="space-between"
+              >
+                <FormLabel
+                  marginBottom="0"
+                  htmlFor="controls"
+                  whiteSpace="nowrap"
+                >
+                  Show control buttons
+                </FormLabel>
+                <Switch defaultChecked={false} id="controls" name="controls" />
+              </FormControl>
+
+              <FormControl
+                display="flex"
+                alignItems="center"
+                justifyContent="space-between"
+              >
+                <FormLabel marginBottom="0" htmlFor="pan" whiteSpace="nowrap">
+                  Allow panning
+                </FormLabel>
+                <Switch defaultChecked={false} id="pan" name="pan" />
+              </FormControl>
+
+              <FormControl
+                display="flex"
+                alignItems="center"
+                justifyContent="space-between"
+              >
+                <FormLabel marginBottom="0" htmlFor="zoom" whiteSpace="nowrap">
+                  Allow zooming
+                </FormLabel>
+                <Switch defaultChecked={false} id="zoom" name="zoom" />
+              </FormControl>
+            </VStack>
+          </form>
+          <Box position="relative" width="100%">
+            <Button
+              size="xs"
+              position="absolute"
+              rounded="false"
```

Comment:
```suggestion
              rounded="none"
```

`false` isn't an option